data/createPages/index.ts

```
@@ -230,8 +250,46 @@ export const createPages: GatsbyNode['createPages'] = async ({ graphql, actions:
+    const languageFiles = (exampleDatum.languages ?? DEFAULT_EXAMPLE_LANGUAGES).reduce((acc, language) => {
+      const filesForLanguage = relatedFiles.reduce<Record<string, string>>((acc, file) => {
+        if (file.language === language && language === 'react' && file.projectRelativePath.startsWith('src/')) {
+          acc[file.projectRelativePath.replace('src/', '')] = file.content;
+        } else if (file.language === language) {
+          acc[file.projectRelativePath] = file.content;
+        }
+        return acc;
+      }, {});
+
+      return {
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)